examples/tokens/token_mint_transaction_fungible.py

```
@@ -127,7 +124,7 @@ def token_mint_fungible(client, token_id, supply_key):
         # Confirm total supply after minting
         info_after = TokenInfoQuery().set_token_id(token_id).execute(client)
         print(f"Total supply after minting: {info_after.total_supply}")
-    except (ValueError, TypeError) as e:
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)